docs/organizations/using-coding-standards.md

```diff
@@ -63,7 +54,42 @@ To create a coding standard for your organization:
     !!! note
         A repository can follow one coding standard at a time. Assigning a coding standard to a repository unassigns any previously assigned coding standard.
 
-    ![Applying the coding standard to repositories](images/coding-standard-apply.png)
+    ![Applying the coding standard to repositories](images/coding-standard-apply.png)<!-- TODO ALA-940 screenshot -->
+
+## Configuring a coding standard with advanced settings {: id="advanced-settings"}
+
+If you need more control when configuring a coding standard, use the advanced settings to more precisely define the tools and code patterns that the coding standard should use.
+
+When creating a new coding standard, you can access the advanced settings from the sensitivity selection page by clicking **Advanced settings**. When editing an existing coding standard, the advanced settings are the default view.
```

Comment:
Tweak, considering the option to customize applies only when creating the standard (it's not an option when editing).

```suggestion
If you need more control when [creating a new coding standard](#creating), use the advanced settings to more precisely define the tools and code patterns that the coding standard should use. You can access the advanced settings from the creation wizard by clicking **Advanced settings** on the group sensitivity step.

When editing an existing coding standard, the advanced settings are the default view.
```